app/mailers/review_added.rb

```diff
@@ -0,0 +1,7 @@
+class ReviewAdded < ApplicationMailer
```

Comment:
Will keep that in mind for the future. For now, to avoid confusion it was agreed to keep it the same